addons/l10n_uy/models/account_move.py

```diff
@@ -22,3 +32,21 @@ def _get_last_sequence_domain(self, relaxed=False):
             where_string += " AND l10n_latam_document_type_id = %(l10n_latam_document_type_id)s"
             param['l10n_latam_document_type_id'] = self.l10n_latam_document_type_id.id or 0
         return where_string, param
+
+    def _get_l10n_latam_documents_domain(self):
+        """ If this is a reversal or debit, suggest only related subtypes, In
```

Comment:
"In master we can skip this by adding new field to the document type to group the e-invoices, e-ticket, e-expo docs" I would not put that in the docstring of the function, you will be able to do the correct change in master in the forward port or in a new Pr, if you do it in the forward port don't hesitate to ping a member of the accounting team :smile: